install_net.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
```

Comment:
This file would not be necessary if we have LabGui on pip, one can then simply run `pip install Labgui`

Reply:
I was unable to have it working properly on pip. Ideally, a seemless installation would have user-prompts for install folder and configuration setup. Using install_net.py will always grab the newest release version of GitHub, as well as initialize a git repository for upgrades and user-modifications, which I believe is very important for LabGUI.

I opened an issue for this if you wish to discuss it further.

Reply:
I am willing to do this, I have never put a package on pip and I would like to learn how to :)
For the time being what you have done is good, let's concentrate on the other issues first.